packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/simulationservice/OptimizeRequestCiemss.java

```diff
@@ -22,7 +22,7 @@ public class OptimizeRequestCiemss implements Serializable {
 	private String modelConfigId;
 	private TimeSpan timespan;
 	@TSOptional
-	private List<OptimizedIntervention> interventions;
+	private OptimizedIntervention interventions;
```

Comment:
This part is a bit confusing since you are intermingling singular and plural variable names.

Can we make the language more consistent? 
If there is a todo like "we need to support a list of interventions some time in the future", please mark a FIXME some where.

Reply:
Fair point. I will change OptimizedIntervention -> OptimizedIntervention**s** in a follow up PR
I will update pyciemss-service too so its up to date